Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -87,8 +107,126 @@ YUI.add('ez-actionbarview-tests', function (Y) {
+            });
+
+            this.view = new Y.eZ.ActionBarView({
+                location: this.location,
+                content: this.content,
+                contentType: this.contentTypeMock
+            });
+        },
+
+        tearDown: function () {
+            delete Y.eZ.CreateContentActionView;
+            delete Y.eZ.TranslateActionView;
+            this.view.destroy();
+        },
+
+        _isActionCreated: function (actionId, testedAttributes) {
+            var actionFound;
+
+            actionFound = Y.Array.find(this.view.get('actionsList'), function (actionView) {
+                return actionView.get('actionId') === actionId;
+            });
+
+            if (!!actionFound) {
+                Y.Object.each(testedAttributes, function (attrValue, attrName) {
+                    Assert.areSame(
+                        attrValue,
+                        actionFound.get(attrName),
+                        'The ' + attrName + ' should have been set to ' + actionId + 'ActionView'
+                    );
+                });
+            }
+
+            Assert.isTrue(
+                !!actionFound,
+                'The ActionBarView should contain ' + actionId + ' action'
+            );
+        },
+
+        _isNotActionCreated: function (actionId, testedAttributes) {
+            var actionFound;
+
+            actionFound = Y.Array.find(this.view.get('actionsList'), function (actionView) {
+                return actionView.get('actionId') === actionId;
+            });
+
+            Assert.isFalse(
+                !!actionFound,
+                'The ActionBarView should not contain ' + actionId + ' action'
+            );
+        },
+
+        "Should instantiate edit buttonActionView": function () {
+            this._isActionCreated('edit', {
+                content: this.view.get('content')
+            });
+        },
+
+        "Should instantiate createContentActionView": function () {
+            this._isActionCreated('createContent', {
+                contentType: this.view.get('contentType')
+            });
+        },
+
+        "Should instantiate translateActionView": function () {
+            this._isActionCreated('translate', {
+                content: this.view.get('content'),
+                location: this.view.get('location')
+            });
+        },
+
+        "Should instantiate moveContentActionView": function () {
+            this._isActionCreated('move', {
+                location: this.view.get('location')
+            });
+        },
+
+        "Should not instantiate sendToTrashActionView": function () {
+            this._isNotActionCreated('sendToTrash', {});
+        },
+
+        "Should instantiate deleteActionView": function () {
+            this._isActionCreated('delete', {});
+        },
     });
 
     Y.Test.Runner.setName("eZ Action Bar View tests");
-    Y.Test.Runner.add(viewTest);
+    Y.Test.Runner.add(viewTest, userViewTest);
```

Comment:
`add` only accepts one parameter so the `userViewTest` is completely ignored, you should add `Y.Test.Runner.add(userViewTest);` instead